data/json/hobbies.json

```diff
@@ -56,6 +56,24 @@
     "traits": [ "DISORGANIZED" ],
     "addictions": [ { "intensity": 20, "type": "alcohol" } ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "sleepingpillhead",
+    "name": "Sleeping Pill Reliance",
+    "description": "You had difficulties falling asleep, so you started taking those sleeping pills. Problem is, now you can not fall asleep without them anymore and the Cataclysm does complicate the matter.",
+    "points": -1,
+    "addictions": [ { "intensity": 10, "type": "sleeping pill" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "caffeine_junkie",
+    "name": "Caffeine Junkie",
+    "description": "Coffee was your day-to-day companion in surviving your job, but getting caffeine into your body will be much harder now.",
+    "points": -1,
```

Comment:
```suggestion
    "points": -1,
    "traits": [ "STIMBOOST" ],
```

Reply:
This trait was basically created for caffeine addicts, and its benefits are mainly for the use of caffeine, so it makes sense to have it with the caffeine addict.